packages/semantic-ui/src/CheckboxWidget/CheckboxWidget.js

```diff
@@ -50,7 +44,7 @@ function CheckboxWidget(props) {
       onBlur={_onBlur}
       onFocus={_onFocus}
       required={required}
-      label={displayLabel ? label || schema.title : false}
```

Comment:
Aren't there reasons for not displaying a label that are still valid? If the above function isn't working properly for semantic UI maybe we fix it?  Also, why not fallback to `schema.title`?

Reply:
schema.title or schema.description because bootstrap 4 uses description?